datadog_checks_base/datadog_checks/base/checks/win/winpdh_base.py

```diff
@@ -206,7 +227,7 @@ def _make_counters(self, key=None, counter_data=([], ())):  # Key left in for re
             else:
                 self._missing_counters.pop((counterset, inst_name, counter_name, dd_name, mtype), None)
 
-            entry = [inst_name, dd_name, m, obj]
+            entry = (inst_name, dd_name, m, obj)
```

Comment:
Small runtime change here, but all this data is private and only handled within this base check, so we _should_ be safe — let's see what the CI tells us.